client/index.html

```diff
@@ -25,7 +25,7 @@
 	<meta name="theme-color" content="#455164">
 
 	</head>
-	<body class="signed-out <%= public ? "public" : "" %>">
+	<body class="signed-out {{ternary public 'public' '' }}">
```

Comment:
Actually yeah, you are right. I don't know why I didn't think of using the proper if helper, that is better. I shall do that.